lnrpc/chainrpc/chainnotifier_server.go

```diff
@@ -88,6 +88,9 @@ type Server struct {
 	started sync.Once
 	stopped sync.Once
 
+	// Required by the grpc-gateway/v2 library for forward compatibility.
+	UnimplementedChainNotifierServer
```

Comment:
nit: move to be the first field

Reply:
Done. Since `sync.Once` is itself a struct this doesn't need to be aligned.